dash/frontend/src/app/app.component.ts

```diff
@@ -43,13 +44,15 @@ export class AppComponent implements OnInit, OnDestroy {
       this.theme = currentTheme;
     });
 
-    timer(0, 10000)
-      .pipe(takeUntil(this.unsubscribe$))
-      .subscribe(() => {
-        if (this.jwtAuthService.isTokenExpired()) {
-          this.router.navigate(['/login']);
-        }
-      });
+    // timer(0, 10000)
```

Comment:
removed the timer that booted you out after a certain amount of time. can uncomment it if you want to keep it.